amd_tressfx/inc/AMD_TressFX.h

```diff
@@ -227,32 +252,80 @@ struct TressFX_Desc
     // Buffer of transformations (one transform per strand) for hair skinning
     // This UAV is used as a structured buffer where each element is a TressFX_HairTransform.
     // The number of elements in the buffer is numTotalHairStrands.
+#ifndef VULKAN
     ID3D11UnorderedAccessView*      pSkinningTransformationsUAV;
+#else
+    VkBufferView                    pSkinningTransformationsUAV;
+#endif
 
     // hair shadow map
+#ifndef VULKAN
     ID3D11ShaderResourceView*       pHairShadowMapSRV;
+#else
+    VkImageView                     pHairShadowMapSRV;
+#endif
 
+#ifndef VULKAN
     ID3D11Device*                   pd3dDevice;
     ID3D11DeviceContext*            pd3dDeviceContext;
     ID3D11ShaderResourceView*       pd3dDepthSRV;
     ID3D11RenderTargetView*         pd3dOutputRTV;
+#else
+    VkDevice                        pvkDevice;
+    uint32_t                        memoryIndexDeviceLocal;
+	uint32_t                        memoryIndexHostVisible;
```

Comment:
Something is up with the indentation. Not too important, but you may want to check for tabs/spaces.